textdb/textdb-exp/src/main/java/edu/uci/ics/textdb/exp/nlp/sentiment/NltkSentimentOperator.java

```diff
@@ -0,0 +1,257 @@
+	    
+	    String[] entrites = new String[2];
+	    entrites[0] = "3NNNNNNNN";
+	    entrites[1] =  "I don't think so.";
+	    csvData.add(entrites);
+	    
+	    String[] entrites2 = new String[2];
+	    entrites2[0] = "6xxxxx";
+        entrites2[1] =  "You are so bad.";
+        csvData.add(entrites2);
+        
+	    writer.writeAll(csvData);
+	    writer.close();
+	    
+	    // call format:
+	    // python3 classifier-loader picklePath dataPath resultPath
+	    List<String> args = new ArrayList<String>(Arrays.asList(PYTHON, PYTHONSCRIPT, PicklePath, BatchedFiles, resultPath));
+//        args.addAll(new ArrayList<String>(Arrays.asList(text)));
+	    System.out.println(args.toString());
+	    ProcessBuilder pb = new ProcessBuilder(args);
+        
+        Process p = pb.start();
+        p.waitFor();
+	    
+	  //Build reader instance
+	    CSVReader reader = new CSVReader(new FileReader(resultPath), SEPARATOR, QUOTECHAR, 1);
+	       
+	      //Read all rows at once
+	    List<String[]> allRows = reader.readAll();
+	       
+	      //Read CSV line by line and use the string array as you want
+	    for(String[] row : allRows){
+//	       System.out.println(Arrays.toString(row));
+	       System.out.println(row[0]+"=="+row[1]);
+//	       System.out.println(Arrays.toString(row).toString().split(", ")[0]+"<-->"+Arrays.toString(row).toString().split(",")[1]);
+	    }
+	}
+	@Override
+	public Tuple getNextTuple() throws TextDBException {
+	    Tuple tuple;
+	    if (tupleBuffer == null){
+	        tupleBuffer = new ArrayList<Tuple>();
+	        //write [ID,text] to a CSV file.
+	        List<String[]> csvData = new ArrayList<>();
+	        
+	        int i = 0;
+	        while (i < predicate.getSizeTupleBuffer()){
+        	    if ((tuple = inputOperator.getNextTuple()) != null) {
+        	        tupleBuffer.add(tuple);
+        	        String[] idText = new String[2];
+        	        idText[0] = tuple.getField(SchemaConstants._ID).getValue().toString();
+        	        idText[1] = tuple.<IField>getField(predicate.getInputAttributeName()).getValue().toString();
+        	        csvData.add(idText);
+        	        i++;
+        	    } else {
+        	        break;
+        	    }
+    	    }
+    	    if (tupleBuffer.isEmpty()) {
+    	        return null;
+    	    }
+    	    try {
+    	        CSVWriter writer = new CSVWriter(new FileWriter(BatchedFiles));
+    	        writer.writeAll(csvData);
+                writer.close();
+            } catch (IOException e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            }
+    	    computeClassLabel(BatchedFiles);
+	    }
+	    
+	    if (cursor == CLOSED) {
+            return null;
+        }
+	    
+	    Tuple outputTuple = tupleBuffer.get(0);
+        tupleBuffer.remove(0);
+        if (tupleBuffer.isEmpty()) {
+            tupleBuffer = null;
+        }
+        
+        List<IField> outputFields = new ArrayList<>();
```

Comment:
These output fields can't be created by only once. Each tuple must be created using different field values.